tools/packaging/packaging.psm1

```diff
@@ -1409,12 +1409,12 @@ function New-UnifiedNugetPackage
 
                 'Microsoft.PowerShell.Commands.Management' {
                     $deps.Add([tuple]::Create([tuple]::Create('id', 'Microsoft.PowerShell.Security'), [tuple]::Create('version', $PackageVersion))) > $null
-                    $deps.Add([tuple]::Create([tuple]::Create('id', 'System.ServiceProcess.ServiceController'), [tuple]::Create('version', '4.4.1'))) > $null
+                    $deps.Add([tuple]::Create([tuple]::Create('id', 'System.ServiceProcess.ServiceController'), [tuple]::Create('version', '4.5.0'))) > $null
```

Comment:
Not related the PR - why we set the versions manually and do not take from csproj files?

Reply:
https://github.com/PowerShell/PowerShell/issues/6978